src/functionalTest/resources/features/F-201 - Assign Access within Organisation/F-201 - Assign Access within Organisation.feature

```diff
@@ -0,0 +1,122 @@
+#=================================================
+@F-201
+Feature: F-201: Assign Access within Organisation
+#=================================================
+
+Background:
+    Given an appropriate test context as detailed in the test data source
+
+#-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+@S-001.1
+Scenario: Solicitor successfully sharing case access with another solicitor in their org (happy path)
+
+    Given a user [S1 - a solicitor, to create a case under their organisation and share it with a fellow solicitor in the same organisation],
+      And a user [S2 - another solicitor in the same organisation, with whom S1 will share a case with an assignment within organisation],
+      And a case [C1, which S1 has just] created as in [Prerequisite_Case_Creation_C1],
+      And a wait time of [5] seconds [to allow for the case just created to appear in search results],
+
+    When a request is prepared with appropriate values,
+      And the request [is to be invoked by S1 to assign access over C1 for S2 within the same organisation],
+      And it is submitted to call the [Assign Access within Organisation] operation of [Manage Case Assignment Microservice],
+
+    Then a positive response is received,
+      And the response has all other details as expected,
+      And a call [by S2 to query his/her case roles granted over C1] will get the expected response as in [S2_Querying_Their_Access_Over_C1].
+
+#-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+@S-001.2
+Scenario: PUI CAA successfully sharing case access with another solicitor in their org (happy path)
+
+    Given a user [S1 - a solicitor, to create a case under their organisation],
+      And a user [S2 - another solicitor in the same organisation, with whom a CAA will share a case with an assignment within organisation],
+      And a user [CAA - a PUI case access admin, to share a case with a solicitor in the same organisation],
+      And a case [C1, which S1 has just] created as in [Prerequisite_Case_Creation_C1],
+      And a wait time of [5] seconds [to allow for the case just created to appear in search results],
+
+    When a request is prepared with appropriate values,
+      And the request [is to be invoked by CAA to assign access over C1 for S2 within the same organisation],
+      And it is submitted to call the [Assign Access within Organisation] operation of [Manage Case Assignment Microservice],
+
+    Then a positive response is received,
+      And the response has all other details as expected,
+      And a call [by S2 to query his/her case roles granted over C1] will get the expected response as in [S2_Querying_Their_Access_Over_C1].
```

Comment:
Should the 2 `S2_Querying_###_Access_Over_C1` guid and files be renamed as `F-201_S2_Querying_###_Access_Over_C1`?